.github/workflows/flake-check.yml

```
@@ -0,0 +1,30 @@
+          nix_path: nixpkgs=channel:nixos-unstable
+
+      - name: Check Root Flake
+        run: nix flake check
+
+      - name: Check Template Flake
+        run: |
+          cd template
+          sed -i 's|url = "github:ClementBobin/FlakeHypr"|url = "path:./"|' flake.nix
+          nix flake check
+          git checkout flake.nix
+          rm -f flake.lock
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)